sources/packages/backend/apps/api/src/services/student-scholastic-standings/student-scholastic-standings.service.ts

```diff
@@ -513,25 +514,45 @@ export class StudentScholasticStandingsService extends RecordDataModelService<St
   }
 
   /**
-   * Get scholastic standing summary details.
+   * Get scholastic standing unsuccessful completion weeks summary details.
    * @param studentId student id to retrieve the scholastic standing summary details.
-   * @return student scholastic standing summary details.
+   * @return student scholastic standing unsuccessful completion weeks summary details.
    */
   async getScholasticStandingSummary(
     studentId: number,
-  ): Promise<ScholasticStandingSummary> {
+  ): Promise<ScholasticStandingSummaryDetails> {
     const studentScholasticStandingSummary = this.repo
       .createQueryBuilder("studentScholasticStanding")
       .select(
         "SUM(studentScholasticStanding.unsuccessfulWeeks)::int",
         "totalUnsuccessfulWeeks",
       )
+      .addSelect("offering.offeringIntensity", "offeringIntensity")
       .innerJoin("studentScholasticStanding.application", "application")
       .innerJoin("application.student", "student")
+      .innerJoin("application.currentAssessment", "assessment")
```

Comment:
As discussed in the dev sync up, application has `offeringIntensity` which can be used now..